src/content/blog/you-probably-don't-need-eslint-config-prettier-or-eslint-plugin-prettier/index.mdx

```diff
@@ -0,0 +1,216 @@
+
+### Why `eslint-config-prettier` Is Often Unnecessary
+
+Over the last few years, ESLint best practices have advanced in two ways (among others):
+
+- ESLint core and most community plugins have determined that enabling overly opinionated rules -especially stylistic ones- in shared configs makes developers dislike ESLint without much real benefit
+- The recommended rulesets in ESLint and typescript-eslint have grown to include most of the beneficial logical rules that shared configs such as `eslint-config-airbnb` were largely used for
+
+As a result, many new projects haven't felt a need to load in opinionated configs such as `eslint-config-airbnb`.
+Many projects start out with a much more straightforward set of configs:
+
+1. To start: `"eslint:recommended"`, ESLint's built-in recommended configuration
+2. If using TypeScript: `"plugin:@typescript-eslint/recommended"` or `"plugin:@typescript-eslint/recommended-type-checked"` for the recommended TypeScript rules
+3. Any framework- or library-specific plugins, such as [`eslint-plugin-jsx-a11y`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y)'s `"plugin:jsx-a11y/recommended"`
+
+**If you don't use a legacy ESLint shared config that enables formatting rules, you probably don't need `eslint-config-prettier`.**
+Adding `eslint-config-prettier` at the end of the `"extends"` list doesn't do anything if nothing enabled formatting rules to begin with.
+Therefore, most projects don't gain any benefit from `eslint-config-prettier`.
+
+Furthermore, two insidious points of confusing can arise from using `eslint-config-prettier` redundantly:
+
+- Seeing a reference to _"prettier"_ in an ESLint configuration can confuse developers new to the area.
+- Nothing stops projects from manually re-enabling formatting rules under the ESLint configuration's `"overrides"` or `"rules"` properties.
+
+I now recommend against including `eslint-config-prettier` in most new projects.
+
+> 💡 Not sure whether you can safely remove `"prettier"` from your `"extends"`?
+> Try removing it, then running `npx eslint-config-prettier some/file.js` to see if it points out any conflicting rules.
+> Running ESLint with [`--print-config`](https://eslint.org/docs/latest/use/command-line-interface#--print-config), can alternately print out the full list of lint rule configurations for a file.
+
+## `eslint-plugin-prettier`
+
+`eslint-plugin-prettier` is an ESLint _plugin_ that provides two things:
+
+- A custom rule, `prettier/prettier`, that runs all of Prettier inside a single ESLint rule
+- A shareable config, `plugin:prettier/recommended`, that enables the `prettier/prettier` rule
+
+For example, in ESLint's legacy config format, you might enable them like:
+
+```json
+{
+	// 1. Load in the prettier/prettier rule from the plugin
+	"plugins": ["prettier"],
+
+	// 2. Turn on the prettier/prettier rule
+	"extends": ["plugin:prettier/recommended"]
```

Comment:
```suggestion
	// 1. Load the prettier/prettier rule and turn it on
	"extends": ["plugin:prettier/recommended"]
```

The recommended config adds `"plugins": ["prettier"]` already - consumers don't need to do that themselves.

Reply:
Ha, I'd noticed this when reading through the eslint-plugin-prettier docs and couldn't believe it at first. What a nice feature. Thanks 😄 https://github.com/typescript-eslint/typescript-eslint/issues/8149